infrastructure/src/main/kotlin/app/discord/repository/jpa/user/schema/UserEntity.kt

```diff
@@ -2,40 +2,46 @@ package app.discord.repository.jpa.user.schema
 
 import jakarta.persistence.*
 import org.hibernate.annotations.CreationTimestamp
-import org.hibernate.annotations.TimeZoneStorage
-import org.hibernate.annotations.TimeZoneStorageType
 import java.time.OffsetDateTime
 
 @Entity(name = "Users")
 class UserEntity(
     @field: Id
     @field: GeneratedValue(strategy = GenerationType.IDENTITY)
-    var id: Long = 0,
+    val id: Long = 0L,
 
     @field: Column(name = "GUILD_ID", nullable = false)
-    var guildId: String,
+    val guildId: String,
 
     @field: Column(name = "USER_ID", nullable = false)
-    var userId: String,
+    val userId: String,
```

Comment:
전체적으로 코드에서 UserId 와 GuildId 를 쌍으로 묶어서 조회하는 로직이 많이 보입니다.
이부분 `UserEntityIdentifier` 라는 [Embedded 객체가 있습니다.](https://github.com/StudyManagerBot/ManagerBot/blob/develop/infrastructure/src/main/kotlin/app/discord/repository/jpa/attendance/schema/UserEntityIdentifier.kt) 변경 고려해보세요.

```suggestion
    @field:Embedded
    val userIdentifier: UserEntityIdentifier,
```
[예제](https://github.com/StudyManagerBot/ManagerBot/blob/develop/infrastructure/src/main/kotlin/app/discord/repository/jpa/attendance/schema/JpaAttendanceHistoryEntity.kt)를 참고해주세요.